tests/test_tenant_unauthorized.py

```diff
@@ -0,0 +1,37 @@
+from unittest.mock import MagicMock
+import pytest
+from assertpy import assert_that
+from dataall.base.api import bootstrap
+from dataall.base.loader import load_modules, ImportMode
+from dataall.base.context import RequestContext, get_context
+from threading import local
+from dataall.base.db.exceptions import TenantUnauthorized
+
+from pytest import fixture
+
+load_modules(modes={ImportMode.API})
+
+CHECK_PERMS = ['archiveOrganization']
+
+
+@fixture(scope='function')
```

Comment:
nit: an alternative and maybe more flexible version of this doesn't require a fixture...

```python
@pytest.mark.parametrize('field_name,resolver', ...)
def test_generic(field_name, resolver, db, client, org1, groupNoTenantPermissions, userNoTenantPermissions):
    local = MagicMock()
    local.context = RequestContext(db, userNoTenantPermissions.username, [groupNoTenantPermissions.groupUri], userNoTenantPermissions)
    with patch('dataall.base.context._request_storage', local):
        # execute test command
```

Reply:
Done in https://github.com/data-dot-all/dataall/pull/1694/commits/75d30c148250894bf791db2bef905c347ed8a3b0